src/Amazon.Common.DotNetCli.Tools/Utilities.cs

```diff
@@ -723,5 +742,44 @@ public static string ReadSecretFromConsole()
             }
             return code.ToString().Trim();
         }
+        
+        
+        public static void CopyDirectory(string sourceDirectory, string destinationDirectory, bool copySubDirectories)
+        {
+            // Get the subdirectories for the specified directory.
+            DirectoryInfo dir = new DirectoryInfo(sourceDirectory);
+
+            if (!dir.Exists)
+            {
+                throw new DirectoryNotFoundException(
+                    "Source directory does not exist or could not be found: "
+                    + sourceDirectory);
+            }
+
+            DirectoryInfo[] dirs = dir.GetDirectories();
```

Comment:
This should be moved down into the if (copySubDirectories) block as you don't use it otherwise.

Reply:
Done